pkg/internal/checkup/checkup_test.go

```diff
@@ -89,14 +89,24 @@ var tests = map[string]struct {
 	expectedErr     string
 }{
 	"noStorageClasses": {
-		clientConfig:    clientConfig{noStorageClasses: true},
-		expectedResults: map[string]string{reporter.DefaultStorageClassKey: checkup.ErrNoDefaultStorageClass},
-		expectedErr:     checkup.ErrNoDefaultStorageClass,
+		clientConfig: clientConfig{noStorageClasses: true, expectNoVMI: true},
+		expectedResults: map[string]string{
+			reporter.DefaultStorageClassKey:   checkup.ErrNoDefaultStorageClass,
+			reporter.PVCBoundKey:              checkup.MessageSkipNoDefaultStorageClass,
+			reporter.VMBootFromGoldenImageKey: checkup.MessageSkipNoDefaultStorageClass,
+			reporter.ConcurrentVMBootKey:      checkup.MessageSkipNoDefaultStorageClass,
+		},
+		expectedErr: checkup.ErrNoDefaultStorageClass,
 	},
 	"noDefaultStorageClass": {
-		clientConfig:    clientConfig{noDefaultStorageClass: true},
-		expectedResults: map[string]string{reporter.DefaultStorageClassKey: checkup.ErrNoDefaultStorageClass},
-		expectedErr:     checkup.ErrNoDefaultStorageClass,
+		clientConfig: clientConfig{noDefaultStorageClass: true, expectNoVMI: true},
+		expectedResults: map[string]string{
+			reporter.DefaultStorageClassKey:   checkup.ErrNoDefaultStorageClass,
+			reporter.PVCBoundKey:              checkup.MessageSkipNoDefaultStorageClass,
+			reporter.VMBootFromGoldenImageKey: checkup.MessageSkipNoDefaultStorageClass,
+			reporter.ConcurrentVMBootKey:      checkup.MessageSkipNoDefaultStorageClass,
+		},
+		expectedErr: checkup.ErrNoDefaultStorageClass,
 	},
```

Comment:
shouldnt you add also virt storage class tests?

Reply:
Yes I should :)